tests/integration/security/authorization_test.go

```diff
@@ -1037,80 +1043,66 @@ func TestAuthorization_Path(t *testing.T) {
-							Target:   a,
-							PortName: "http",
-							Scheme:   scheme.HTTP,
-							Path:     path,
-						},
-					},
-					ExpectAllowed: expectAllowed,
-				}
-			}
-			cases := []rbacUtil.TestCase{
-				newTestCase("/public", true),
-				newTestCase("/private", false),
-				newTestCase("/public/../private", false),
-				newTestCase("/public/./../private", false),
-				newTestCase("/public/.././private", false),
-				newTestCase("/public/%2E%2E/private", false),
-				newTestCase("/public/%2e%2e/private", false),
-				newTestCase("/public/%2E/%2E%2E/private", false),
-				newTestCase("/public/%2e/%2e%2e/private", false),
-				newTestCase("/public/%2E%2E/%2E/private", false),
-				newTestCase("/public/%2e%2e/%2e/private", false),
-			}
-
+			ns := apps.Namespace3
 			args := map[string]string{
 				"Namespace": ns.Name(),
 			}
 			policies := tmpl.EvaluateAllOrFail(t, args,
 				file.AsStringOrFail(t, "testdata/authz/v1beta1-path.yaml.tmpl"))
 			ctx.Config().ApplyYAMLOrFail(t, ns.Name(), policies...)
 			defer ctx.Config().DeleteYAMLOrFail(t, ns.Name(), policies...)
-
-			rbacUtil.RunRBACTest(t, cases)
+			for _, srcCluster := range ctx.Clusters() {
+				ctx.NewSubTest(fmt.Sprintf("In %s", srcCluster.Name())).Run(func(ctx framework.TestContext) {
+					f := apps.F.Match(echo.InCluster(srcCluster).And(echo.NameSpace(ns.Name())))
+					e := apps.E.Match(echo.NameSpace(ns.Name()))
+					callCount := 1
+					if ctx.Clusters().IsMulticluster() {
+						// so we can validate all clusters are hit
+						callCount = util.CallsPerCluster * len(ctx.Clusters())
+					}
+
+					newTestCase := func(path string, expectAllowed bool) rbacUtil.TestCase {
+						return rbacUtil.TestCase{
+							Request: connection.Checker{
+								From: f[0],
+								Options: echo.CallOptions{
+									Target:   e[0],
+									PortName: "http",
+									Scheme:   scheme.HTTP,
+									Path:     path,
+									Count:    callCount,
+								},
+								DestClusters: e.Clusters(),
+							},
+							ExpectAllowed: expectAllowed,
+						}
+					}
+					cases := []rbacUtil.TestCase{
+						newTestCase("/public", true),
+						newTestCase("/private", false),
+						newTestCase("/public/../private", false),
+						newTestCase("/public/./../private", false),
+						newTestCase("/public/.././private", false),
+						newTestCase("/public/%2E%2E/private", false),
+						newTestCase("/public/%2e%2e/private", false),
+						newTestCase("/public/%2E/%2E%2E/private", false),
+						newTestCase("/public/%2e/%2e%2e/private", false),
+						newTestCase("/public/%2E%2E/%2E/private", false),
+						newTestCase("/public/%2e%2e/%2e/private", false),
+					}
+					rbacUtil.RunRBACTest(ctx, cases)
+				})
+			}
 		})
 }
 
 // TestAuthorization_Audit tests that the AUDIT action does not impact allowing or denying a request
 func TestAuthorization_Audit(t *testing.T) {
 	framework.NewTest(t).
 		Run(func(ctx framework.TestContext) {
+			if ctx.Clusters().IsMulticluster() {
+				ctx.Skip()
```

Comment:
same here, why skip the test? and we need better comment and AI on this.

Reply:
Same above.